content/posts/wasi-experimental-http.md

```diff
@@ -0,0 +1,146 @@
+---
+title:
+  "An experimental outbound HTTP library for the WebAssembly System Interface"
+description:
+  "Send HTTP requests from Rust and AssemblyScript Wasm modules running in
+  Wasmtime"
+date: 2021-03-08 00:00:00 +0000 UTC
+authorname: "Radu Matei"
+author: "@matei_radu"
+authorlink: "https://twitter.com/matei_radu"
+image: "images/twitter-card.png"
+tags: ["wasm", "wasi"]
+---
+
+Over the last year, our team has been experimenting with executing WebAssembly
+workloads on the server using the [WebAssembly System Interface][wasi] and
+[Wasmtime][wasmtime], and while more and more [proposals][proposals] get
+implemented, networking is one scenario that doesn't have a stable API yet,
```

Comment:
break the sentence: "on the server using WASI and wasmtime. While more and more proposalsare being implemented, networking is one scenario that doesn't have a stable API yet. This limits...."

too many phrases to parse easily.